spec/test-utils/test_indexeddb_cryptostore_dump/no_cached_msk_dump/index.ts

```diff
@@ -256,5 +273,6 @@ export const MSK_NOT_CACHED_DATASET: DumpDataSetInfo = {
     keyQueryResponse: KEY_QUERY_RESPONSE,
     rotatedKeyQueryResponse: ROTATED_KEY_QUERY_RESPONSE,
     backupResponse: BACKUP_RESPONSE,
+    newBackupResponse: NEW_BACKUP_RESPONSE,
```

Comment:
Since this is just a different backup key which is unrelated to the rest of `MSK_NOT_CACHED_DATASET` (right?), I don't really think this belongs here.

Hardcode it as a const in the `describe` block in `spec/integ/crypto/rust-crypto.spec.ts`?

Reply:
It is related, the signature is correct